### PR TITLE
Move up to resident form

### PR DESCRIPTION
**Pull request by kosiakkatrina**
**What**  
Move the screen view to resident form when sumbit is clicked and residentInfo is empty
Check all of the residentInfo fields before displaying the summary form

### CODE REVIEW

File: components/Feature/SupportSummary/index.js [hunk 1]
```diff
@@ -22,10 +22,22 @@ const SupportSummary = ({
...
-    if (!residentInfo && hideForm) {
+  const toggleDetail = e => {
+    if (
+      (!residentInfo.firstName ||
+        !residentInfo.lastName ||
+        !residentInfo.phone ||
+        !residentInfo.email ||
+        !residentInfo.address ||
+        !residentInfo.postcode ||
+        !residentInfo['date-of-birth-year'] ||
```
Comment (elena-vi) [inline components/Feature/SupportSummary/index.js:33]: dob is not mandatory

**Review by elena-vi — APPROVED** [review]
besides the dob looks good

**Review by Duslerke — APPROVED** [review]
Are all of those resident fields required for submission?

**Comment by kosiakkatrina** [comment]
> Are all of those resident fields required for submission?

Yup! They're required for both referrals and conversations

**Pull request merged**